lib/improver/tests/utilities/cube_manipulation/test__equalise_cube_attributes.py

```diff
@@ -88,119 +88,53 @@ def test_cubelist_no_history_removal(self):
 
         cube1 = self.cube_ukv.copy()
         cube2 = self.cube.copy()
-        cube1.attributes.update({'grid_id': 'ukx_standard_v1'})
-        cube2.attributes.update({'grid_id': 'unknown_grid'})
-
-        cubelist = iris.cube.CubeList([cube1, cube2])
-
-        result = _equalise_cube_attributes(cubelist)
-
-        self.assertIn("grid_id", result[0].attributes.keys())
-        self.assertEqual(result[0].attributes["grid_id"],
-                         'ukx_standard_v1')
-        self.assertIn("grid_id", result[1].attributes.keys())
-        self.assertEqual(result[1].attributes["grid_id"],
-                         'unknown_grid')
-
-    def test_cubelist_title_identical(self):
-        """Test that the utility does nothing to title if they match"""
-
-        cube1 = self.cube_ukv.copy()
-        cube2 = self.cube.copy()
-        cube1.attributes.update({'title':
-                                 'Operational UKV Model Forecast'})
-        cube2.attributes.update({'title':
-                                 'Operational UKV Model Forecast'})
+        cube1.attributes.update({'mosg__model_configuration':
+                                 'uk_det'})
+        cube2.attributes.update({'mosg__model_configuration':
+                                 'uk_det'})
 
         cubelist = iris.cube.CubeList([cube1, cube2])
 
         result = _equalise_cube_attributes(cubelist)
-        self.assertEqual(result[0].attributes["title"],
-                         result[1].attributes["title"])
-        self.assertEqual(result[0].attributes["title"],
-                         'Operational UKV Model Forecast')
+        self.assertEqual(result[0].attributes["mosg__model_configuration"],
+                         result[1].attributes["mosg__model_configuration"])
+        self.assertEqual(result[0].attributes["mosg__model_configuration"],
+                         'uk_det')
 
-    def test_cubelist_title(self):
+    def test_cubelist_mosg__model_configuration(self):
         """Test that the utility adds coords for model if not matching"""
 
         cube1 = self.cube_ukv.copy()
         cube2 = self.cube.copy()
-        cube1.attributes.update({'title':
-                                 'Operational UKV Model Forecast'})
-        cube2.attributes.update({'title':
-                                 'Operational Mogreps UK Model Forecast'})
+        cube1.attributes.update({'mosg__model_configuration':
+                                 'uk_det'})
+        cube2.attributes.update({'mosg__model_configuration':
+                                 'uk_ens'})
 
         cubelist = iris.cube.CubeList([cube1, cube2])
 
         result = _equalise_cube_attributes(cubelist)
 
         self.assertArrayAlmostEqual(result[0].coord("model_id").points,
-                                    np.array([0]))
-        self.assertEqual(result[0].coord("model").points[0],
-                         'Operational UKV Model Forecast')
+                                    np.array([3000]))
```

Comment:
As I understand it, it doesn't really matter what the numbers are, they're only used internally. They can be modified though and I think that spacing of 1000s means that it is unfeasable for duplicate values to be attained.

Reply:
For a dimension coordinate you need a coordinate with a dimension hence model_id is a value (so the coordinate can be put in an order). The values were originally arbitrary i.e. first model = 1000, second model = 2000. The reason that they are multiples of 1000 is so that if we have realizations on the cube we can still merge the cubes together. A new coordinate model_realization is added which is model_id + realization number (deterministic models also have a realization coord with realization number =0 set). The model_id needed to be big enough that a unique model_realization number would be created regardless of how many ensemble members we had. The current implementation means we can have up to 999 ensemble members before we run into a problem.

Reply:
Thanks Caroline, that makes sense.